src/Core/Geographic/Coordinates.js

```diff
@@ -18,6 +19,9 @@ export function ellipsoidSizes() {
 
 const ellipsoid = new Ellipsoid(ellipsoidSizes());
 
+proj4.defs('EPSG:3946',
```

Comment:
I'm not sure I want to wrap `proj4` at all.
I think I'd prefer the user to directly call `proj4.defs(...)` and transform `supportedCRS` to a function capable of querying `proj4` definition.